components/style/properties/helpers/animated_properties.mako.rs

```diff
@@ -2104,6 +2271,18 @@ impl Animatable for MatrixDecomposed3D {
     }
 }
 
+impl ComputeSquaredDistance for MatrixDecomposed3D {
```

Comment:
This implementation can be derived if you implement that trait for `Quaternion`, which you should instead of having an intrinsic `distance` method.